kythe/go/extractors/validation/validation.go

```diff
@@ -62,6 +65,9 @@ type Coverage struct {
 	Paths []Dir
 }
 
+// Stats on contents of archive
```

Comment:
Worth putting expected key space(s) in here, or just leave it freeform?

Reply:
Done